gui/build.py

```diff
@@ -79,6 +79,28 @@ def do_bundle():
     for dirname in ('build', 'dist'):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
+    # create a intaller
+    python_lib_path = os.path.join(
+        os.getenv('MOZREGUI_PYTHONPATH', "C:\\Python27"),
+        "Lib"
+    )
+    makensis_path = os.path.join(
+        os.getenv('MOZREGUI_PYTHONPATH', "C:\\NSIS"),
+        "makensis.exe"
+    )
+    gui_path = os.path.realpath('.')
+    moz_path = os.path.realpath('..')
+    python_dir = os.path.dirname(sys.executable)
+    args = []
+    args.append('--icon=wininst/app_icon.ico')
+    args.append('--base-name=Win32GUI')
+    args.append('--include-path=%s;%s;%s\site-packages;%s'
+                % (gui_path, moz_path, python_lib_path, python_lib_path))
+    args.append('--target-name=mozregression-gui.exe')
+    args.append('--target-dir=dist')
+    args.append('mozregui/main.py')
+    call(sys.executable, os.path.join(python_dir, 'cxfreeze'), *args)
+    call(makensis_path, 'wininst.nsi', cwd='wininst')
 
     # run pyinstaller
     pyinstaller = py_script('pyinstaller')
```

Comment:
Hm, It's my fault for forget to remove it. Actually, I had change the code when I work under the windows but I need switch to Linux and change my code again before I commit and push them.
So, a lot mistakes happened because of my carelessness.....:(